src/components/TaxonDisplay.tsx

```diff
@@ -14,18 +28,26 @@ interface TaxonDisplayProps {
 }
 
 /*
- * Main component Function.
+ * Main component Function. displays taxon ID and name
  */
 const TaxonDisplay = (props: TaxonDisplayProps): JSX.Element => {
   const { currentTaxon, taxonID } = props;
 
+  //HOOKS here
+  const classes = useStyles();
+
+  //RETURN ELEMENT HERE
   return (
     <Card>
-      <CardContent>
-        <Typography variant="h5" component="h2">
+      <CardContent className={classes.containerClass}>
+        <Typography
+          className={classes.taxonNameClass}
+          variant="h5"
+          component="h2"
+        >
           Current Taxon: {currentTaxon}
         </Typography>
-        <Typography color="textSecondary" gutterBottom>
+        <Typography className={classes.taxonIdClass} gutterBottom>
```

Comment:
gutterBottom may also be possible to include in makeStyles. Try `gutterBottom: true` ?